applications/teletype/src/teletype_templates.erl

```diff
@@ -163,20 +163,38 @@ templates_source(_TemplateId, 'undefined', _ResellerId) ->
     'undefined';
 templates_source(TemplateId, AccountId, AccountId) ->
     case fetch_notification(TemplateId, AccountId) of
-        {'ok', Template} -> kz_doc:account_id(Template);
+        {'ok', Template} ->
+            templates_source_has_attachments(TemplateId, AccountId, AccountId, Template);
         {'error', 'not_found'} -> ?KZ_CONFIG_DB;
         {'error', _E} -> 'undefined'
     end;
 templates_source(TemplateId, AccountId, ResellerId) ->
     case fetch_notification(TemplateId, AccountId) of
         {'error', 'not_found'} ->
-            lager:debug("failed to find template ~s in account ~s", [TemplateId, AccountId]),
-            ParentId = teletype_util:get_parent_account_id(AccountId),
-            templates_source(TemplateId, ParentId, ResellerId);
-        {'ok', Template} -> kz_doc:account_id(Template);
+            parent_templates_source(TemplateId, AccountId, ResellerId);
+        {'ok', Template} ->
+            templates_source_has_attachments(TemplateId, AccountId, ResellerId, Template);
         {'error', _E} -> 'undefined'
     end.
 
+-spec templates_source_has_attachments(ne_binary()
+                                      ,ne_binary()
+                                      ,ne_binary()
+                                      ,kz_json:object()) -> api_binary().
```

Comment:
@fenollp happy to minimize it. I was aiming to keep an 80-or-less character line length. Is there any style standard when it comes to specs (or anything else for matter) that I can adhere to?